pkg/terraform/store.go

```diff
@@ -49,9 +49,10 @@ type ProviderConfiguration map[string]interface{}
 // Setup holds values for the Terraform version and setup
 // requirements and configuration body
 type Setup struct {
-	Version       string
-	Requirement   ProviderRequirement
-	Configuration ProviderConfiguration
+	Version        string
+	Requirement    ProviderRequirement
+	Configuration  ProviderConfiguration
+	CredentialsEnv []string
```

Comment:
I felt a bit confused about having this as a []string rather than some map[string]string keeping key-value pairs but looks like this is already the type how cmd.Env expects the env vars. So, looking good.